tarantool/connection_pool.py

```diff
@@ -750,6 +756,12 @@ def eval(self, expr, *args, mode=None):
         :param mode: Request mode.
         :type mode: :class:`~tarantool.Mode`
 
+        :param on_push: Refer to
+            :paramref:`~tarantool.Connection.call.params.on_push`.
```

Comment:
```suggestion
            :paramref:`~tarantool.Connection.eval.params.on_push`.
```

and, please, check others too

Reply:
Done